test/acceptance_with_python/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Check if Python is installed
+if ! command -v python3 &>/dev/null; then
+    echo "Python is not installed. Please install Python and try again."
+    exit 1
+fi
+
+# Check if a virtual environment (venv) exists
+if [ ! -d "venv" ]; then
+    echo "Creating a new virtual environment (venv)..."
+    python3 -m venv .venv
+fi
+
+# Activate the virtual environment
+source .venv/bin/activate
+
+cd "$SCRIPT_DIR" || return
+
+pip install --upgrade pip --quiet
+pip install -r requirements.txt --quiet
```

Comment:
can we change this to `pip3`? or it doesn't matter? (I have always used `pip3`)

Reply:
it doesn't matter because we activated the virtual env :)